code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1049,20 +1049,35 @@
 	..()
 	return TRUE
 
-/datum/reagent/medicine/corazone
-	//Corazone no longer stops heart attacks. look into C2 penthrite.
-	name = "Corazone"
-	description = "A medication used to treat pain, fever, and inflammation, along with failing livers."
-	color = "#F49797"
+/datum/reagent/medicine/higadrite
+	name = "Higadrite"
+	description = "A medication utilized to treat ailing livers."
+	color = "#FF3542"
 	self_consuming = TRUE
 
-/datum/reagent/medicine/corazone/on_mob_add(mob/living/M)
+/datum/reagent/medicine/higadrite/on_mob_add(mob/living/M)
 	..()
 	ADD_TRAIT(M, TRAIT_STABLELIVER, type)
 
-/datum/reagent/medicine/corazone/on_mob_end_metabolize(mob/living/M)
+/datum/reagent/medicine/higadrite/on_mob_end_metabolize(mob/living/M)
+	..()
 	REMOVE_TRAIT(M, TRAIT_STABLELIVER, type)
+
+/datum/reagent/medicine/cordiolis_hepatico
+	name = "Cordiolis Hepatico"
+	description = "A strange, pitch-black reagent that seems to absorb all light. Effects unknown."
+	color = "#000000"
+	self_consuming = TRUE
+
+/datum/reagent/medicine/higadrite/on_mob_add(mob/living/M)
```

Comment:
when you copy+pasted this (and the bit below this), you forgot to replace "higadrite" with "cordiolis_hepatico"

Reply:
> Higadrite is a rock name. You are following rock and stone naming convention. I'm not a fan of the existing name scheme for chems, but this DOES spit in the face of the rock scientists.

chems with -rite suffix work by synthesizing into kidney stones with various effects